src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.sex !== 'f' && (
```

Comment:
I guess you have to check if `age` exists then ` && (`

also you wrote just hardcoded value `I am 37`, use `age`  variable here